### ISSUE TITLE
Wolf cannot be tamed

### ISSUE THREAD

**Issue by dsharp1** [Nukkit-coders/MobPlugin#109]
### Info

This is:
- [X ] bug report
- [ ] feature request
- [ ] other

### Your issue / suggestion
From the source code, it looks like the only mob that is tameable is the Wolf.  I was thinking of trying to contribute a tameable cat.  But while experimenting with the Wolf, I could never get the Wolf to trigger.  I used the Bone as the source code describes, but the Wolf never interacts.

### Configuration
Link to debugpaste:
```
https://hastebin.com/awegoqosec
```
Config:
```
config-version: 8

entities:
  autospawn-ticks: 200
  despawn-ticks: 72000
  worlds-spawning-disabled: "exampleworld1, exampleworld2"

autospawn:
  bat: 2
  blaze: 2
  chicken: 2
  cod: 2
  cow: 2
  creeper: 2
  dolphin: 2
  donkey: 2
  enderman: 2
  ghast: 2
  husk: 2
  horse: 2
  magmacube: 2
  mooshroom: 2
  ocelot: 2
  parrot: 2
  pig: 2
  polarbear: 2
  pufferfish: 2
  rabbit: 2
  salmon: 2
  sheep: 2
  skeleton: 2
  slime: 2
  spider: 2
  squid: 2
  stray: 2
  tropicalfish: 2
  turtle: 2
  witch: 2
  witherskeleton: 2
  wolf: 2
  zombie: 2
  zombiepigman: 2
```

**Comment by ZizzyZizzy** [comment]
This works for me in v1.9.0.5, but the wolf doesn't follow you. He will eventually show hearts when given bones, and then sits when you click on him though.

**Comment by PetteriM1** [comment]
Fixed in b0268fd
